test-data/unit/python2eval.test

```diff
@@ -152,13 +156,16 @@ f(**params)
 
 [case testFromFutureImportUnicodeLiterals2_python2]
 from __future__ import unicode_literals
-def f(x: str) -> None: pass
+def f(x):
+    # type: (str) -> None
```

Comment:
This (like a few others) was needed because we're now running these tests with `--fast-parser`, and the fast PY2 parser rejects PY3-style annotations!